package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "serve": "vite preview --port 8080",
+    "preview": "vite preview",
```

Comment:
I wasted an hour trying to figure out why I couldn't load this UI using `npm run serve`, which serves up a previous local build (npm run build) if there is one.

`npm run serve` is still needed for Namerequest UI since it still uses Vue CLI instead of Vite. This can lead to develop confusion when they work on multiple repos at once.

Reply:
Credit where credit is due: this solution, which I love, was Karim's idea 😀

Reply:
Thank you Sev! ❤️